server/executor/trigger/http.go

```diff
@@ -199,3 +199,70 @@ func mapResp(resp *http.Response) model.HTTPResponse {
 		Body:       body,
 	}
 }
+
+func (t *httpTriggerer) Variables(ctx context.Context, test model.Test, executor expression.Executor) (expression.VariablesMap, error) {
+	triggerVariables := expression.VariablesMap{}
+	http := test.ServiceUnderTest.HTTP
+
+	if http == nil {
+		return triggerVariables, fmt.Errorf("no settings provided for HTTP triggerer")
+	}
+
+	urlVariables, err := executor.StatementTermsByType(WrapInQuotes(http.URL, "\""), expression.EnvironmentType)
+
+	if err != nil {
+		return triggerVariables, err
+	}
+	triggerVariables = triggerVariables.MergeStringArray(urlVariables)
```

Comment:
Same suggestion as the gRPC endpoint:

```suggestion
	triggerVariables := expression.VariablesMap{}
	
	http := test.ServiceUnderTest.HTTP
	if http == nil {
		return triggerVariables, fmt.Errorf("no settings provided for HTTP triggerer")
	}

	urlVariables, err := executor.StatementTermsByType(WrapInQuotes(http.URL, "\""), expression.EnvironmentType)
	if err != nil {
		return triggerVariables, err
	}
	
	triggerVariables = triggerVariables.MergeStringArray(urlVariables)
```